packages/parser/src/language/pie/pieValueConverter.ts

```diff
@@ -0,0 +1,50 @@
+import type { CstNode, GrammarAST, ValueType } from 'langium';
+import { DefaultValueConverter } from 'langium';
+
+import { CommonValueConverter } from '../common/commonValueConverters.js';
+
+export class PieValueConverter extends DefaultValueConverter {
+  protected override runConverter(
+    rule: GrammarAST.AbstractRule,
+    input: string,
+    cstNode: CstNode
+  ): ValueType {
+    let value: ValueType | undefined = CommonValueConverter.customRunConverter(
+      rule,
+      input,
+      cstNode
+    );
+    if (value === undefined) {
+      value = PieValueConverter.customRunConverter(rule, input, cstNode);
+    }
+
+    if (value === undefined) {
+      return super.runConverter(rule, input, cstNode);
+    } else {
+      return value;
+    }
+  }
+
+  /**
+   * A method contains convert logic to be used by class itself or `MermaidValueConverter`.
+   *
+   * @param rule - Parsed rule.
+   * @param input - Matched string.
+   * @param _cstNode - Node in the Concrete Syntax Tree (CST).
+   * @returns converted the value if it's pie rule or `null` if it's not.
+   */
+  public static customRunConverter(
+    rule: GrammarAST.AbstractRule,
+    input: string,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    _cstNode: CstNode
+  ): ValueType | undefined {
+    if (rule.name === 'PIE_SECTION_LABEL') {
+      return input
+        .replace(/"/g, '')
+        .trim()
+        .replaceAll(/[\t ]{2,}/gm, ' ');
+    }
```

Comment:
Why is this required?

Reply:
The `.replace(/"/g, '')` part is essential because the rule matches everything between double quotes including the wrapper quotes, so we manually have to remove it.

The other two lines are to clean the what is between doable quotes, the current render does that:

```
pie showData
    title dsfdsf
    "fdsfdf": 1
    "            fdffdsfd      ":1
    "                          ": 1  
```

```mermaid
pie showData
    title dsfdsf
    "fdsfdf": 1
    "            fdffdsfd      ":1
    "                          ": 1  
```

Reply:
I think we should also remove newlines since they're also removed in layout.

Reply:
`.replaceAll(/[\t ]{2,}/gm, ' ')` I don't think we should be doing this. Trimming the ends is fine, but we should not be changing things inside the user input.

Reply:
Then what should `expect` in test cases here?

Reply:
Done 69198ccccd8d4082461d3fbd45d43de8bb05f70e